packages/instrumentation-langchain/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@traceloop/instrumentation-langchain",
-  "version": "0.5.0",
+  "version": "0.5.4-alpha.0",
```

Comment:
same

Reply:
This will get auto updated by lerna when I release